src/_guides/whats-new.md

```diff
@@ -22,11 +22,29 @@ and follow the [Dart blog][].
 This section lists notable changes made from May 12, 2022,
 through August 30, 2022.
 For details about the 2.18 release,
-see [Dart 2.18: ][], and the [change log][https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180].
+see [Dart 2.18: Objective-C & Swift interop][],
+and the [SDK changelog][].
+
+[Dart 2.18: Objective-C & Swift interop]: https://medium.com/dartlang/dart-2-18-f4b3101f146c
+[SDK changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180
 
 In addition to bug fixes and incremental improvements,
 we made the following changes to this site:
 
+* Added a workaround to [Fixing common type problems][], 
+  for the rare case where type inference might still incorrectly infer an argument type is null.
```

Comment:
**suggestion:** Simplify the copy and try not to use the same word twice in the same sentence. Also, changed to active voice.

```suggestion
* Added a workaround to [Fixing common type problems][]. 
  This corrects for when type inference might be conclude that an argument type can use a null value.
```